pkgs/development/python-modules/csv2md/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "lzakharov";
+    repo = "csv2md";
+    rev = "v${version}";
+    hash = "sha256-INwZRbyJmyDd5SV7PBdqUc/3FO91upTgaxBfP/Giaw4=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [ "csv2md" ];
+
+  meta = {
+    description = "Command line tool for converting CSV files into Markdown tables";
```

```suggestion
    changelog = "https://github.com/lzakharov/csv2md/releases/tag/v${version}";
    description = "Command line tool for converting CSV files into Markdown tables";
```

Maybe we could add meta.changelog here.
What do you think?